tests/e2e/example.spec.ts

```diff
@@ -1,88 +1,109 @@
+  suite: 'Main Example Group',
+})('Example test with minimal metadata', async (t) => {
   await t
     .typeText('#developer-name', 'John Smith')
     .click('#submit-button')
     .expect(Selector('#article-header').innerText)
     .eql('Thank you, John Smith!');
 });
 
-fixture('TestCafé example test fixture 2').page('http://devexpress.github.io/testcafe/example').meta({
-  severity: Severity.CRITICAL,
-  suite: 'FixtureGroup',
-});
+test
+  .meta({
+    suite: 'Main Example Group',
+  })
+  .skip('Example skipped test', async (t) => {
+    await t
+      .typeText('#developer-name', 'John Smith')
+      .click('#submit-button')
+      .expect(Selector('#article-header').innerText)
+      .eql('Thank you, Jane Smith!');
+  });
 
 test.meta({
-  severity: Severity.MINOR,
-})('My third e2e test', async (t) => {
+  suite: 'Failing Test Example Group',
+})('Example failing e2e test without steps', async (t) => {
   await t
     .typeText('#developer-name', 'John Smith')
     .click('#submit-button')
     .expect(Selector('#article-header').innerText)
-    .eql('Thank you, John Smith!');
+    .eql('Thank you, Jane Smith!');
 });
 
 test.meta({
-  severity: Severity.BLOCKER,
-})('My fourth e2e test', async (t) => {
+  suite: 'Failing Test Example Group',
+})('Example failing e2e test with steps', async (t) => {
+  await step('Add developer name to form', t, t.typeText('#developer-name', 'John Smith'));
+  await step(
+    'Submit form and check result',
+    t,
+    t.click('#submit-button').expect(Selector('#article-header').innerText).eql('Thank you, Jane Smith!'),
+  );
+});
+
+test.meta({
+  suite: 'Flaky Test Example Group',
+  description: 'Does, however, require that TestCafé runs in Quarantine mode.',
+})('Actual flaky test example', async (t) => {
+  const random: number = Math.random();
+  await t.expect(random).gte(0.5);
+});
+
+test.meta({
+  suite: 'Flaky Test Example Group',
+  flaky: true,
+})('Manual flaky test example', async (t) => {
   await t
     .typeText('#developer-name', 'John Smith')
     .click('#submit-button')
     .expect(Selector('#article-header').innerText)
     .eql('Thank you, John Smith!');
 });
 
-test('My failing e2e test', async (t) => {
-  await t
-    .typeText('#developer-name', 'John Smith')
-    .click('#submit-button')
-    .expect(Selector('#article-header').innerText)
-    .eql('Thank you, Jane Smith!');
+fixture('TestCafé Example Fixture 2').page('http://devexpress.github.io/testcafe/example').meta({
+  severity: Severity.CRITICAL,
+  suite: 'Example Fixture Group',
 });
 
-test.skip('My skipped e2e test', async (t) => {
+test.meta({
+  severity: Severity.MINOR,
+})('Example severity metadata override MINOR', async (t) => {
   await t
     .typeText('#developer-name', 'John Smith')
     .click('#submit-button')
     .expect(Selector('#article-header').innerText)
-    .eql('Thank you, Jane Smith!');
+    .eql('Thank you, John Smith!');
 });
 
-test('My actual flaky e2e test', async (t) => {
```

Comment:
We need more certainty that the flaky test is flaky. (Read remove false positives).
Can we make an assertion on a persisted variable that is updated the first 2 attempts and that will pass the assertion the third time?

Reply:
Reworked the not-consistently-flaky Flaky test to be more consistent in it's flakiness. (By adding a persisted variable that is increased by one each run and only matches on the third run)